app/components/ui/files/publish-modal/component.js

```diff
@@ -231,7 +235,6 @@ export default Ember.Component.extend({
             Called when the `Publish` button is clicked. It controls the flow
             of logging in and communicating with the `createPackage` endpoint.
             */
-
            let self = this;
            if (self.get('publishingFinish') == true) {
```

Comment:
@ThomasThelen - we should ALWAYS use the less permissive `===` for comparisons. Also, in this case, to check if the expression is _truthy_, you would have just needed to do `if (self.get('publishingFinish')) { ... } `